lib/erl_interface/src/Makefile.in

```diff
@@ -683,6 +690,8 @@ release: opt
 	$(INSTALL_DIR) "$(RELSYSDIR)/src/prog"
 	$(INSTALL_DIR) "$(RELEASE_PATH)/usr/include"
 	$(INSTALL_DIR) "$(RELEASE_PATH)/usr/lib"
+	$(INSTALL_DIR) "$(DESTDIR)/$(libdir)/pkgconfig"
+	$(INSTALL_DATA) $(PKG-CONFIG) "$(DESTDIR)/$(libdir)/pkgconfig"
```

Comment:
These are problematic. It is possible to install OTP using paths determined by `configure` (and then this will work), but also using an explicitly given path at install time. This either using `make release RELEASE_ROOT=<PATH>` or `./otp_build release [-a] <PATH>`. When giving the path at install time this will install `erl_ei.pc` at a completely different place than where OTP is installed.

A solution could be to only install `erl_ei.pc` when installing OTP using `make install` at the top. Info indicating that we are installing using the `configure`d path then needs to be passed down to this make file. This can for example be done via a make variable.

Reply:
Alright, I got it.